### ISSUE TITLE
Set formItemLayout, but displaying form as vertical.

### ISSUE THREAD

**Issue by danielcai1987** [rekit/antd-form-builder#6]
Dear supnate,

Thank you for sharing this antd-form-builder!

I'm trying to adjust the column width using formItemLayout.
As in the API Reference, the default value of formItemLayout is [8, 16]. I tried to set it as
```
const meta = {
      formItemLayout:[6, 18],
      fields: [{...}]
}
```

Or

```
const meta = {
     formItemLayout:{
        labelCol: { span: 8 },
        wrapperCol: { span: 16 },
      },
      fields: [{...}]
}
```

But the form layout will change to vertical, not displaying as horizontal form.

Could you give a working example of set the column width?

Thank you very much!

**Comment by supnate** [comment]
Antd form has three layout options: horizontal, vertical, inline , `inline` layout is what you need. https://ant.design/components/form/#components-form-demo-layout

```jsx
<Form layout="inline">
  <FormBuilder meta={meta} form={form}/>
</Form>
```

**Comment by danielcai1987** [comment]
@supnate Hi Nate, thank you for your prompt reply.
I'd like to keep using the **horizontal** layout. But only change the default column width from [8, 16] to [6, 18]. Is it possible?

Currently I cannot get the expected result, the form generated is changing the layout to vertical.

**Comment by supnate** [comment]
Sorry that I misunderstood your requirement. It's really a bug of form builder. Now fixed by 90a030f6a2cf13e8ee665c16d4610ef47f7ba3d0 . You can update to `1.0.8` to fix it.

Thanks for reporting this!

**Comment by supnate** [comment]
The fix is: ab9e6134145077bcb56a0d72ababb971955c0eae
